diff --git a/examples/iotlab/00-configuration/tsch-project-conf.h b/examples/iotlab/00-configuration/tsch-project-conf.h
index d090878..953e94f 100644
--- a/examples/iotlab/00-configuration/tsch-project-conf.h
+++ b/examples/iotlab/00-configuration/tsch-project-conf.h
@@ -77,7 +77,7 @@
 /* TSCH logging. 0: disabled. 1: basic log. 2: with delayed
  * log messages from interrupt */
 #undef TSCH_LOG_CONF_LEVEL
-#define TSCH_LOG_CONF_LEVEL 1
+#define TSCH_LOG_CONF_LEVEL 2
 
 /* 6TiSCH minimal schedule length.
  * Larger values result in less frequent active slots: reduces capacity and saves energy. */
@@ -86,7 +86,7 @@
 
 /* IEEE802.15.4 PANID */
 #undef IEEE802154_CONF_PANID
-#define IEEE802154_CONF_PANID 0xcdef
+#define IEEE802154_CONF_PANID 0x0006
 
 #undef TSCH_CONF_JOIN_MY_PANID_ONLY
 #define TSCH_CONF_JOIN_MY_PANID_ONLY 1
