diff --git a/examples/iotlab/04-er-rest-example/er-example-server.c b/examples/iotlab/04-er-rest-example/er-example-server.c
index b78dc78..a3211e3 100644
--- a/examples/iotlab/04-er-rest-example/er-example-server.c
+++ b/examples/iotlab/04-er-rest-example/er-example-server.c
@@ -156,7 +156,7 @@ PROCESS_THREAD(er_example_server, ev, data)
 /*  rest_activate_resource(&res_chunks, "test/chunks"); */
 /*  rest_activate_resource(&res_separate, "test/separate"); */
   rest_activate_resource(&res_push, "test/push");
-/*  rest_activate_resource(&res_event, "test/serial"); */
+  rest_activate_resource(&res_event, "test/serial");
 /*  rest_activate_resource(&res_sub, "test/sub"); */
 /*  rest_activate_resource(&res_b1_sep_b2, "test/b1sepb2"); */
 #if PLATFORM_HAS_LEDS
